calico-enterprise/multicluster/create-a-managed-cluster/create-a-managed-cluster-helm.mdx

```diff
@@ -66,9 +66,11 @@ If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubern
 
 1. Add any other customizations you require to `values.yaml`. To see values that can be customized in the chart, see the [helm docs](https://helm.sh/docs/) or run the following command:
```

Comment:
I was wondering if the helm docs link (seen here and in the managed cluster page) is correct? I was expecting this to link to a page that was specific to our chart